packages/strapi-plugin-sentry/admin/src/index.js

```diff
@@ -0,0 +1,35 @@
+// NOTE TO PLUGINS DEVELOPERS:
```

Comment:
I don't think you need to add any admin folder in the plugin for now

Reply:
I added this based on @soupette 's suggestion

It's only so that it shows up in the plugins list page
<img width="1440" alt="Screenshot 2020-12-16 at 12 59 12" src="https://user-images.githubusercontent.com/8087692/102474489-ff551d80-4058-11eb-8348-801bd284a387.png">

Reply:
Ok why not